assets/texts/es/amoSummary.txt

```diff
@@ -1,3 +1,3 @@
-Esta extensión te permite generar rápidamente códigos QR offline de una URL abierta en tus pestañas o cualquier texto (seleccionado)! 👍
+Esta extensión te permite generar rápidamente códigos QR de la URL de la pestaña abierta o cualquier otro texto! 👍
 
-Funciona completamente offline protegiendo así tu privacidad[TOO LONG: y cuenta con un vasto rango de funciones incluyendo la opción de colorear tus códigos QR]!
+Funciona completamente offline, protegiendo así tu privacidad!
```

Comment:
Yep, the comma is an improvement. On the other hand, I think the exclamation mark does not belong here anymore (it was part of what was removed). A final period would also suit better the seriousness of the point made.